examples/webapp/README.md

```diff
@@ -0,0 +1,21 @@
+# A minimal web app example for yapapi
+
+This example demonstrates using yapapi's Services API to run a web application composed of two services deployed to separate provider hosts and each one using its own vm image.
+
+## Components
+
+### DB backend: rqlite
+
+For the backend, we decided to use **[rqlite](https://github.com/rqlite/rqlite)** - an "easy-to-use, lightweight, distributed relational database, which uses SQLite as its storage engine."
+
+One reason to use `rqlite` is that it's very easy to set-up and another, more important one is that its clustering capabilities make it extremely useful for distributed deployments on Golem.
```

Comment:
```suggestion
One reason to use `rqlite` is that it's very easy to set-up, and another - more important one - is that its clustering capabilities make it extremely useful for distributed deployments on Golem.
```
Just punctuation :)

But a more important thing is: this strongly suggests there is a way to use this in a distributed way (on the current Golem). Are we sure about this?

Reply:
I believe it is possible, just _might_ require a specially-prepared image